ArkKit/ark-game/view/ui-kit/ArkUIKitViewController.swift

```diff
@@ -49,9 +72,15 @@ extension ArkUIKitViewController: GameStateRenderer {
                                                     canvasFrame: canvasContext.canvasFrame)
         canvas.render(using: canvasRenderer, to: canvasContext)
     }
+    
+    func onScreenResize(_ delegate: @escaping ScreenResizeDelegate) {
```

Comment:
We can move this function to the section where `ArkUIKitViewController: AbstractView`